.gitignore

```
@@ -0,0 +1,2 @@
+/target
+/.idea
```

Оставь просто .idea и доавь ещё несколько
```suggestion
.idea
*.swp
.env*
```